packages/compass-e2e-tests/index.ts

```diff
@@ -49,6 +49,8 @@ const FIRST_TEST = 'tests/time-to-first-query.test.ts';
 let compassWeb: ChildProcessWithoutNullStreams;
 
 async function setup() {
+  debug('X DISPLAY', process.env.DISPLAY);
```

Comment:
To debug `"xvfb-maybe --auto-servernum --server-args=\"-screen 0 1432x840x24\" -- ts-node index.ts`